models/playlists.js

```diff
@@ -34,6 +35,15 @@ class Playlists {
                 })
         .returning(['id', 'title', 'created_at', 'updated_at']);
   }
+
+  async removeFavorite(playlistId, favoriteId){
+    return database('favorites_playlist')
+          .where({
```

Comment:
Nice use of "where" here, I did not know that you could put multiple attributes where!